DEVPROD-952: Spruce Hosts page docs (#7402)
evergreen-ci/evergreen: docs/Hosts/Spawn-Hosts.md

If you'd like to get a notification before a host expires, you can [set up a
notification](../Project-Configuration/Notifications.md#spawn-host-expiration) for it.

## Hosts Page

The spruce hosts page shows all hosts and offers three batch actions applicable to hosts:

> 1. Update Status
>    You can force a state change to these statuses:

Reviewer: ```suggestion
1. Update Status

   You can force a state change to these statuses:
```
needs an extra line break for the intended formatting

>    -Decomissioned: Terminate a host running tasks after it's done running its current task.

Reviewer: ```suggestion
   - Decommissioned: Terminate a host after it's done running its current task.
```

>    -Quarantined: Stop a host from running tasks without terminating it or shutting it down.

Reviewer: ```suggestion
   - Quarantined: Stop a host from running tasks without terminating it.
```
(q: what makes this different from the plain "Stopped"? It is worth noting here or down below if possible)

>    -Terminate: Actually terminate the host.

Reviewer: ```suggestion
   - Terminate: Shut down the host.
```

>    -Stopped: Stop it.

Reviewer: ```suggestion
   - Stopped: Stop the host.
```

>    -Runnning: Start it back up.

Reviewer: ```suggestion
   - Running: Start the host.
```

> 2. Restart Jasper
>    Try forcing the Evergreen agent (which runs in a system process called Jasper) to start back up in a way that's less disruptive than just rebooting the computer.

Reviewer: ```suggestion
2. Restart Jasper

   This option will try forcing the Evergreen agent (which runs in a system process called Jasper) to start back up in a way that's less disruptive than just rebooting the host.
```
needs an extra line break for the intended formatting

> 3. Reprovision
>    Hosts need to have a few starter files on the file system before they can run tasks, and pressing that button gives them the files they need if they're a static host(because there's no way to auto-populate them with the files they need). This option is useful for static hosts that get extremely borked (like the file system is corrupted and it loses all the data).

Reviewer: ```suggestion
3. Reprovision

   Hosts need to have a few starter files on the file system before they can run tasks. Sometimes static hosts can get into bad states (e.g. the file system is corrupted) and stop functioning correctly. Reprovisioning a host will repopulate these files for static hosts.
```
needs an extra line break for the intended formatting & changed some wording here
